src/share/export.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="UTF-8">
+	<title>Exporting</title>
+</head>
+
+<body style="height: 100%; width: 100%;">

```

Comment:
Move any CSS you need to an external file vs. using `style` attributes on elements.